Classes/FrontEnd/SelectorWidget.php

```diff
@@ -217,13 +218,30 @@ private function collectPlaces()
             return;
         }
 
-        $dataOfPlaces = \Tx_Oelib_Db::selectMultiple(
-            'tx_seminars_sites.*',
-            'tx_seminars_sites, tx_seminars_seminars_place_mm',
-            'tx_seminars_sites.uid = tx_seminars_seminars_place_mm.uid_foreign ' .
-            'AND tx_seminars_seminars_place_mm.uid_local IN (' .
-            $this->seminarBag->getUids() . ')'
-        );
+        /** @var ConnectionPool $connectionPool */
+        $connectionPool = GeneralUtility::makeInstance(ConnectionPool::class);
+        $queryBuilder = $connectionPool->getQueryBuilderForTable('tx_seminars_sites');
+        $dataOfPlaces = $queryBuilder
+            ->select('tx_seminars_sites.*')
+            ->from('tx_seminars_sites')
+            ->join(
+                'tx_seminars_sites',
+                'tx_seminars_seminars_place_mm',
+                'mm',
+                $queryBuilder->expr()->eq(
+                    'mm.uid_foreign',
+                    $queryBuilder->quoteIdentifier('tx_seminars_sites.uid')
+                )
+            )
+            ->where(
+                $queryBuilder->expr()->in(
+                    'mm.uid_local',
+                    $this->seminarBag->getUids()
```

Comment:
`in()` should have an array of values as the second argument. So we'll need `GeneralUtiltity::intExplode()` here.

Reply:
`in()` takes an array or a string as arguments. If the argument is a string like `1,2,3`, the method transforms this into `(1,2,3)`. 

I tested this with:

```
->where(
    $queryBuilder->expr()->in(
        'mm.uid_local',
        '1,3,5'
    )
)
```

Reply:
Sounds okay. Thanks for the research!